front-end/src/lib/Footer/Footer.component.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import "./Footer.style.scss";
+
+const Footer: React.FC = () => (
+  <footer>
+    <img src="./PLACEHOLDER" alt="Catan In Space Logo" />
+    <div className="links">
+      <a href="/home" aria-label="Home Page">Home</a>
+      <a href="/contact" aria-label="Contact Page">Contact</a>
+      <a href="/how-to-play" aria-label="How To Play Page">How To Play</a>
+      <a href="/policy/privacy" aria-label="Privacy Policy Page">Privacy Policy</a>
+      <a href="/policy/TOS" aria-label="Terms Of Service Page">Terms of Service</a>
+      <a href="/careers" aria-label="Careers Page">Careers</a>
+    </div>
+    <div className="socials">
+
+      <a href="https://www.facebook.com" aria-label="www.Facebook.com">
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 448" fill="none">
```

Comment:
For readability, I don't think it's a bad idea to have each SVG element as its own component.

Like:
- facebook-logo.component.tsx
- x-logo.component.tsx

Then we can use them in other places if needed 😄

Reply:
That's something I can setup! Where would you like these components to live?